### PR TITLE
Adding Support for VPN Connections

### PR DESCRIPTION
**Pull request by EarthmanT**
support create, delete, describe

### CODE REVIEW

File: moto/ec2/utils.py [hunk 1]
```diff
@@ -193,6 +197,14 @@ def vpc_ids_from_querystring(querystring_dict):
     return vpc_ids
 
 
+def vpn_connection_ids_from_query_string(querystring_dict):
```
Comment (spulec) [inline moto/ec2/utils.py]: I think we can use the `sequence_from_querystring` below instead.

**Comment by spulec** [comment]
Could you add a couple tests?

**Comment by EarthmanT** [comment]
@spulec added tests

**Comment by spulec** [comment]
thanks!

**Pull request merged**